plugin/pkg/global/resourcereferencemanager/admission.go

```diff
@@ -138,7 +139,10 @@ func (r *ReferenceManager) SetInternalCoreInformerFactory(f coreinformers.Shared
 	controllerDeploymentInformer := f.Core().InternalVersion().ControllerDeployments()
 	r.controllerDeploymentLister = controllerDeploymentInformer.Lister()
 
-	readyFuncs = append(readyFuncs, seedInformer.Informer().HasSynced, shootInformer.Informer().HasSynced, cloudProfileInformer.Informer().HasSynced, secretBindingInformer.Informer().HasSynced, quotaInformer.Informer().HasSynced, projectInformer.Informer().HasSynced, controllerDeploymentInformer.Informer().HasSynced)
+	exposurClassInformer := f.Core().InternalVersion().ExposureClasses()
```

Comment:
As discussed, let's use the `v1beta1` lister here.

Reply:
You mean the `v1alpha1` lister here? The `v1beta1` version does not have the `exposureClass` anymore. Done.